src/IO/AssimpLoader/AssimpGeometryDataLoader.hpp

```diff
@@ -1,12 +1,13 @@
 #ifndef RADIUMENGINE_ASSIMP_GEOMETRY_DATA_LOADER_HPP
 #define RADIUMENGINE_ASSIMP_GEOMETRY_DATA_LOADER_HPP
 
-#include <set>
-
 #include <Core/File/DataLoader.hpp>
 #include <Core/Math/LinearAlgebra.hpp>
 #include <IO/RaIO.hpp>
 
+#include <memory>
+#include <set>
+
```

Comment:
why are these needed here ?

Reply:
`set` was already here, and `memory` is here for `unique_ptr`.

Reply:
OK. But I've strange compilation warning (and sometime errors). I investigate ...